dependencyManagement/build.gradle.kts

```diff
@@ -12,17 +12,19 @@ val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
 // this line is managed by .github/scripts/update-sdk-version.sh
-val otelVersion = "1.19.0"
+val otelSdkVersion = "1.19.0"
+// this line is managed by .github/scripts/update-instrumentation-version.sh
+val otelInstrumentationVersion = "1.19.0"
```

Comment:
intentionally left this at 1.19.0 to see if dependabot picks it up

... and if it doesn't, then I'll use it to test out updated workflow